hydra-node/json-schemas/api.yaml

```
@@ -1643,12 +1709,21 @@ components:
       type: integer
       minimum: 0
 
-    RawTransaction:
-      title: RawTransaction
-      description: |
-        A CBOR-serialised signed Cardano transaction, encoded in base16.
-      type: string
-      contentEncoding: base16
+    Transaction:
+      type: object
+      required:
+        - cborHex
```

Must: complete this to be usable as a schema to test for cardano-cli compatibility

```suggestion
        - type
        - description
        - cborHex
```

As commented above, we do care about that integration and this schema is used in the relevant endpoints / websocket messages. Hence, this schema should capture the requirements of `cardano-cli transaction` commands on the data given to it.

This allows us to test against this schema and use this as a proxy for testing compatibility (the alternative being integration tests, but those are slow)